### ISSUE TITLE
Get XPO ready for curation

### ISSUE THREAD

**Issue by matentzn** [obophenotype/xenopus-phenotype-ontology#47]
- [x] #46 
- [ ] #39 
- [ ] #45

**Comment by matentzn** [comment]
@seger can you link all the tickets here that absolutely need to happen before we XPO can go into the XenBase curation tools?

**Comment by seger** [comment]
#48, #41, #40

Term additions - #69, #66, #64, #58 ,#54, #53, #50, #39, #38, #37

Term obsoletions (almost done) - #68, #67, #65, #63, #62, #61, #60, #59, #57, #56, #55, #52, #51, #49, #45, #44, #43, #42

#26 - New terms have been added. Erik - break out the discussion bits into new tickets for future consideration, then close.

#22, #4, #3 - Maybe already done?

**Comment by seger** [comment]
- [ ] #48
- [ ] #41
- [ ] #40

Term additions
- [ ] #69
- [ ] #66
- [ ] #64
- [ ] #58
- [ ] #54
- [ ] #53
- [ ] #50
- [ ] #39
- [ ] #38
- [ ] #37

Term obsoletions (almost done)
- [ ] #68
- [ ] #67
- [ ] #65
- [ ] #63
- [ ] #62
- [ ] #61
- [ ] #60
- [ ] #59
- [ ] #57
- [ ] #56
- [ ] #55
- [ ] #52
- [ ] #51
- [ ] #49
- [ ] #45
- [ ] #44
- [ ] #43
- [ ] #42

Maybe already done?
- [ ] #22
- [ ] #4
- [ ] #3

**Comment by seger** [comment]
- [ ] #26 - New terms have been added. Erik - break out the discussion bits into new tickets for future consideration, then close.
